pkg/models/runtime_env_label.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package models
+
+import (
+	"fmt"
+
+	"openpitrix.io/openpitrix/pkg/utils"
+)
+
+const RuntimeEnvLabelTableName = "runtime_env_label"
+
+func NewRuntimeEnvLabelId() string {
+	return utils.GetUuid("rel-")
+}
+
+type RuntimeEnvLabel struct {
+	RuntimeEnvLabelId string
+	RuntimeEnvId      string
+	LabelKey          string
+	LabelValue        string
+}
+
+var RuntimeEnvLabelColumns = GetColumnsFromStruct(&RuntimeEnvLabel{})
+
+func NewRuntimeEnvLabel(runtimeEnvId, labelKey, labelValue string) *RuntimeEnvLabel {
+	return &RuntimeEnvLabel{
+		RuntimeEnvLabelId: NewRuntimeEnvLabelId(),
+		RuntimeEnvId:      runtimeEnvId,
+		LabelKey:          labelKey,
+		LabelValue:        labelValue,
+	}
+}
+
+func RuntimeEnvLabelToString(runtimeEnvLabel *RuntimeEnvLabel) string {
```

Comment:
runtime_env_label should return as a struct.
reference: https://github.com/runzexia/openpitrix/commit/a9a1fdbb97d7d92d1af526f58dab42b1cb86087b#r28152438